masterlist.yaml

```diff
@@ -324,6 +356,14 @@ plugins:
         itm: 69
         udr: 8
         nav: 1
+    clean:
+      # Version 1.59.39.0 - Auto Clean
+      - crc: 0x5eb9eade
+        util: SSEEdit v3.2.1
+  - name: 'SkyrimVR.esm'
+    global_priority: -127
```

Comment:
VR again, remove?

Reply:
Nope, Skyrim VR uses the same masterlist because there's so little difference, and `SkyrimVR.esm` needs to load early.